integrations/flink_connector/flink-connector-timestream/src/test/java/com/amazonaws/samples/connectors/timestream/SinkInitContext.java

```diff
@@ -60,6 +60,11 @@ public int getNumberOfParallelSubtasks() {
         return 0;
     }
 
+    @Override
+    public int getAttemptNumber() {
+        return 1;
```

Comment:
Why return is constant and always 1?

Reply:
I checked previous version (Before update the flink to 1.7 the default value was 1 for attempts) and in new version we don't have default value.(Need to override interface).

Reply:
Let's add a code comment explaining why 1 is a default value.

Reply:
Comment has been added